### PR TITLE
Outgoing webhook with Adaptive card

### PR DESCRIPTION
**Pull request by Trinetra-MSFT**
Added Adaptive card support for outgoing Web Hook sample code:
1. Used the existing webhook bot sample
2. Added Adaptive card support trigger
3. Language used: C#

### Testing

image

### CODE REVIEW

File: samples/outgoing-webhook/csharp/README.md [hunk 1]
```diff
@@ -1,18 +1,64 @@
...
+- Visual Studio
+
+- Asp.net Core
+
+## To try this sample
+
+> Note these instructions are for running the sample on your local machine, the tunnelling solution is required because
+
+1) Clone the repository
+    git clone https://github.com/RamaMohanaChary/microsoft-teams-sample-outgoing-webhook-master
```
Comment (Wajeed-msft) [inline samples/outgoing-webhook/csharp/README.md:41]: Please correct this to point to new repo.

File: samples/outgoing-webhook/csharp/WebhookSampleBot/Web.config [hunk 2]
```diff
@@ -4,12 +4,15 @@
   http://go.microsoft.com/fwlink/?LinkId=301879
   -->
 <configuration>
-  <appSettings></appSettings>
+  <appSettings>
+    <add key="MS_OutgoingWebHookSecretToken" value="uDGKhpdsY6eSyIRdYEueTfeO2ftlMOEmi9+eJ7RaocM=" />
```
Comment (Wajeed-msft) [inline samples/outgoing-webhook/csharp/WebhookSampleBot/Web.config:8]: Please remove the value from config file. If this is unused then please remove it.

**Review by Wajeed-msft — CHANGES_REQUESTED** [review]

File: samples/outgoing-webhook/csharp/README.md (hunk 1 above)
Comment (Trinetra-MSFT) [reply samples/outgoing-webhook/csharp/README.md]: Fixed this review comment

File: samples/outgoing-webhook/csharp/WebhookSampleBot/Web.config (hunk 2 above)
Comment (Trinetra-MSFT) [reply samples/outgoing-webhook/csharp/WebhookSampleBot/Web.config]: Removed value from web.config

**Review by Wajeed-msft — APPROVED** [review]

File: samples/outgoing-webhook/csharp/README.md [hunk 3]
```diff
@@ -1,18 +1,64 @@
...
+    - Navigate to `samples/outgoing-webhook/csharp` folder
+    - Select `WebhookSampleBot.sln` file
+    - Press `F5` to run the project
+
+3) Run ngrok - point to port 3978
+    ```bash
+    ngrok http -host-header=rewrite 3978
+    ```
+4)  __*This step is specific to Teams.*__
+    - **Edit** the `manifest.json` contained in the  `teamsAppManifest` folder to replace your Microsoft App Id (that was created when you registered your tab earlier) *everywhere* you see the place holder string `<<YOUR-MICROSOFT-APP-ID>>` (depending on the scenario the Microsoft App Id may occur multiple times in the `manifest.json`)
```
Comment (Wajeed-msft) [inline samples/outgoing-webhook/csharp/README.md:55]: Outgoing webhooks do not need app manifest. Please correct these steps.

**Review by Wajeed-msft — CHANGES_REQUESTED** [review]

File: samples/outgoing-webhook/csharp/README.md (hunk 3 above)
Comment (Trinetra-MSFT) [reply samples/outgoing-webhook/csharp/README.md:55]: Okay

**Review by Wajeed-msft — APPROVED** [review]

**Pull request merged**